src/main/java/blitz/gameoflife/RleParser.java

```diff
@@ -95,8 +84,39 @@ public int[][] decodeData(String content) {
         return grid;
     }
 
-    public int[][] loadGridFromRleFile(String filePath) throws IOException {
-        String content = readFile(filePath);
+    public int[][] getGridFromClipboard(String content) {
```

Comment:
You don't care where this text comes from, it's not necessary to put it in the name of the method.